rasa/core/processor.py

```diff
@@ -1007,13 +1011,17 @@ def _log_action_on_tracker(
             isinstance(event, ActionExecutionRejected) for event in events
         )
         if not action_was_rejected_manually:
-            logger.debug(f"Policy prediction ended with events '{prediction.events}'.")
+            structlogger.debug(
+                "processor.actions.log", prediction_events=prediction.events
```

Comment:
```suggestion
                "processor.actions.policy_prediction", prediction_events=prediction.events
```
not sure if this log is even usefull given that we don't log the policy they belong to...